addons/web/static/src/js/views/calendar/calendar_renderer.js

```diff
@@ -171,10 +171,12 @@ return AbstractRenderer.extend({
     template: "CalendarView",
     config: {
         CalendarPopover: CalendarPopover,
+        calendarBoxTemplate: 'calendar-box',
     },
     custom_events: _.extend({}, AbstractRenderer.prototype.custom_events || {}, {
         edit_event: '_onEditEvent',
         delete_event: '_onDeleteEvent',
+        render_event: '_renderEvents',
```

Comment:
The changes about the configurable template and the ones about the custom render event have to be in separate commits since they do not depend on each other.

Sorry if that was unclear.